app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -146,25 +159,25 @@ export class AddForm extends React.Component {
             orderNumber,
         },
       }, () => {
-        this.props.addDraftOrder(this.state.draftOrder);
+        addDraftOrderAction(this.state.draftOrder);
         if (this.state.draftOrder.action === 'NEW') {
-          this.props.setOrderAction('DRAFT', this.state.orderNumber);
+          setOrder('DRAFT', this.state.orderNumber);
         } else {
-          this.props.setOrderAction('DRAFT_EDIT', this.props.orderNumber);
+          setOrder('DRAFT_EDIT', this.props.orderNumber);
```

Comment:
you can destructure orderNumber here.

Reply:
there is a state 'orderNumber' and props 'orderNumber', that's y I didnt do the distructuring here here

Reply:
If that is the case you should create a ticket to handle the naming while also destructing. Because this can also be refactored further.

Reply:
Or you can still refactor it here since that is what the ticket is about.

Reply:
That makes sense. Let me take care of it.

Reply:
Done